RawLamb/LamellaPlacement.cs

```diff
@@ -208,8 +208,54 @@ public XmlElement ToXml(XmlDocument doc)
             plane.AppendChild(xaxis);
             plane.AppendChild(yaxis);
             main.AppendChild(plane);
-                
+
             return main;
         }
+
+        public LamellaPlacement FromXml(XmlElement element)
```

Comment:
1. `FromXml()` needs to be a static function
2. Would be nice to clean up white space